### PR TITLE
Accordion checklist and panel react components

### PR DESCRIPTION
**Pull request by stanvishne**

### CODE REVIEW

**Review by ArielDKenan — APPROVED** [review]
great job

File: src/react/Checklist.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from './Checkbox.js';
+
+const Checklist = ({ items = [], className, onChange }) => (
+    <div className={className}>
+        {items.map((item, index) => {
+            return (
+                <div key={`checkbox-item-${index}`} className='checkbox-item'>
```
Comment (ArielDKenan) [inline src/react/Checklist.js:9]: what's the point of this div?

Comment (ArielDKenan) [reply src/react/Checklist.js:9]: I see, it's aligning with ng2

**Comment by islavi** [comment]
Please add also class for the title, so we can style it.
Also can you add position of the arrow: right/left

**Pull request merged**